### PR TITLE
Persist hero properties with a GitHub gist

### PR DESCRIPTION
**Pull request by ghost**
Heros' properties can now be persisted to a JSON file, saved as a GitHub gist.  Use a hero's public certificate as a key.

  This could be used as a base for an accounting or karma system (issue #3).

  A next step could also include a way for users to "claim" a hero key (via signing) to associate the key to a friendly name, which could make it possible/more convenient to give/claim karma.

### CODE REVIEW

**Comment by cinderblock** [comment]
I don't fully understand what you're doing with this or how it's intended to be used. Could you please elaborate?

**Comment by ZainRizvi** [comment]
I like the idea, but the code is a little hard to follow :)

It would be great to turn this into a generic data store
